google-cloud-debugger/test/google/cloud/debugger/breakpoint/evaluator_test.rb

```diff
@@ -45,6 +45,10 @@ def mutating_func2
   $global_var = 2
 end
 
+def infinite_loop
+  infinite_loop()
```

Comment:
The current detection mechanism uses line event callback, so it only works if the infinite loop calls executes some code in each iteration. It can't detect a loop on a single line. Changed this test case to use a multi-line loop.